cmd/arena/commands/top_node.go

```diff
@@ -57,16 +59,32 @@ func NewTopNodeCommand() *cobra.Command {
 				os.Exit(1)
 			}
 			nd := newNodeDescriber(client, allPods)
-			nodeInfos, err := nd.getAllNodeInfos()
+			nodeInfos, gpushareNodes, err := nd.getAllNodeInfos()
```

Comment:
Please don't add `gpushareNodes` which may confuse others.  I think `isGpushare` is an attribute of `nodeInfo`.